lib/multitimer/src/lib.rs

```diff
@@ -213,8 +213,8 @@ mod fakes {
     use core::cell::Cell;
 
     thread_local! {
-        pub static CURRENT_TIME: Cell<u64> = Cell::new(0);
-        pub static TIMER_SETTING: Cell<(Option<u64>, u32)> = Cell::default();
+        pub static CURRENT_TIME: Cell<u64> = const { Cell::new(0) };
+        pub static TIMER_SETTING: Cell<(Option<u64>, u32)> = const { Cell::new((None, 0)) };
```

Comment:
(turns out we're not running clippy on multitimer, so these lines had warnings.)